node/flatpak-node-generator.py

```diff
@@ -851,6 +851,10 @@ def child_url(self, child: str) -> str:
 
     def find_binaries(self, binary: str) -> Iterator['ElectronBinaryManager.Binary']:
         for electron_arch, flatpak_arch in self.ELECTRON_ARCHES_TO_FLATPAK.items():
+            # Electron v19+ drop linux-ia32 support.
+            if version_tuple(self.version) >= version_tuple("19.0.0") and electron_arch == "ia32":
```

Comment:
We already have a custom version parser.
```suggestion
            if SemVer.parse(self.version) >= SemVer.parse("19.0.0") and electron_arch == "ia32":
```

Reply:
Done. Please check [b60bc6f](https://github.com/flatpak/flatpak-builder-tools/pull/282/commits/b60bc6f0fcc6f761c979c558726ffd53f20afbd3). Thanks.